dids/src/test/kotlin/web5/sdk/dids/DidKeyTest.kt

```diff
@@ -1,10 +1,10 @@
+package web5.sdk.dids
+
 import org.junit.jupiter.api.Test
-import web5.dids.DidKey
 
 class DidKeyTest {
   @Test
   fun `it works`() {
-    val result = DidKey.create()
-    println(result.first)
+    DidKey.create()
```

Comment:
fix unit test + many otther unit tests (in another PR)

Reply:
fixed